src/vs/vscode.d.ts

```diff
@@ -4916,25 +4931,28 @@ declare module 'vscode' {
 		 *
 		 * *Note* that only files within the current [workspace folders](#workspace.workspaceFolders) can be watched.
 		 *
-		 * @param globPattern A glob pattern that is applied to the names of created, changed, and deleted files.
+		 * @param globPattern A glob pattern (either absolute like `**∕*.{ts,js}` or a [relative pattern](#RelativePattern))
+		 * that is applied to the names of created, changed, and deleted files.
 		 * @param ignoreCreateEvents Ignore when files have been created.
 		 * @param ignoreChangeEvents Ignore when files have been changed.
 		 * @param ignoreDeleteEvents Ignore when files have been deleted.
 		 * @return A new file system watcher instance.
 		 */
-		export function createFileSystemWatcher(globPattern: string, ignoreCreateEvents?: boolean, ignoreChangeEvents?: boolean, ignoreDeleteEvents?: boolean): FileSystemWatcher;
+		export function createFileSystemWatcher(globPattern: string | RelativePattern, ignoreCreateEvents?: boolean, ignoreChangeEvents?: boolean, ignoreDeleteEvents?: boolean): FileSystemWatcher;
 
 		/**
 		 * Find files in the workspace.
 		 *
 		 * @sample `findFiles('**∕*.js', '**∕node_modules∕**', 10)`
-		 * @param include A glob pattern that defines the files to search for.
-		 * @param exclude A glob pattern that defines files and folders to exclude.
+		 * @param include A glob pattern (either absolute like `**∕*.{ts,js}` or a [relative pattern](#RelativePattern))
+		 * that defines the files to search for.
+		 * @param exclude A glob pattern (either absolute like `**∕*.{ts,js}` or a [relative pattern](#RelativePattern))
+		 * that defines files and folders to exclude.
 		 * @param maxResults An upper-bound for the result.
 		 * @param token A token that can be used to signal cancellation to the underlying search engine.
 		 * @return A thenable that resolves to an array of resource identifiers.
 		 */
-		export function findFiles(include: string, exclude?: string, maxResults?: number, token?: CancellationToken): Thenable<Uri[]>;
+		export function findFiles(include: string | RelativePattern, exclude?: string | RelativePattern, maxResults?: number, token?: CancellationToken): Thenable<Uri[]>;
```

Comment:
To avoid repetition we could consider something like 

```ts
/**
 * Write Jsdoc once
 */
type GlobPattern = string | RelativePattern

```

Reply:
👍

Reply:
@jrieken done via https://github.com/Microsoft/vscode/pull/34695/commits/0ba6fb6e4ea5cf8dde5396a061e6bacb057f2367